client/components/mobile/ActionStrip.jsx

```diff
@@ -38,7 +38,7 @@ const ActionStrip = ({ actions }) => (
 
 ActionStrip.propTypes = {
   actions: PropTypes.arrayOf(PropTypes.shape({
-    icon: PropTypes.any,
+    icon: PropTypes.any, //eslint-disable-line
```

Comment:
Is there a way to fix this so that you don't have to disable linting on this line?

Reply:
@catarak I think in order to resolve it we have to explicitly specify the data type of icon field.
Can you please tell me which data type does it use?

Reply:
It should be `PropTypes.component`! I fixed it 😄